db/institutions.yml

```diff
@@ -482,6 +495,32 @@ lehighu:
   name: Lehigh University
   state: PA
   zip: '18015'
+losalamosnationallaboratory:
+  aka:
+    - LANL
+  city: Los Alamos
+  country: USA
+  day: 7
+  month: 2
+  name: Los Alamos National Laboratory
+  state: New Mexico
+  updated: 2020-02-07 16:33:04.900594
+  uuid: a95d6075-d238-4611-afe5-d05520d8280e
+  year: 2020
+  zip: '87545'
+losalamosneutronsciencecenter:
```

Comment:
this should be a department of lanl.  We may have to change the contact to make it work, but please put as a department in lanl